### DISCUSSION TITLE
Any easier way to upload Inventree Logo and/or Favicon

### DISCUSSION THREAD

**Discussion by E-Hain** [inventree/InvenTree#5467] [Ideas]
It would be really nice to have an admin setting that allows for easier uploading of a logo or favicon. To be quite honest I haven't been able to figure out how to use my own favicon. Is that currently possible?

**Comment by SchrodingersGat** [comment]
You can customise the logo and the splash screen - see the [docs here](https://docs.inventree.org/en/latest/start/config/?h=custom#customisation-options). Currently I do not think that the favicon can be adjusted.

It would be a good addition to enable these custom images to be uploaded from the interface, but currently has to be done as part of system setup

**Reply by E-Hain** [reply]
Yeah, I feel that the implementation of it wouldn't be too hard. You could have a file created at setup that remains empty like

```/where-ever-data-is-mounted/static/img/custom-img/logo```

And then another for favicons

```/where-ever-data-is-mounted/static/img/custom-img/favicon```

Then, on start up, if a toggle like "custom logo" and/or "custom favicon" are toggled on, then the server checks those files for any usable logo and/or favicon. If nothing exists in those files or isn't usable, throw an error message. In any case, that custom isn't toggled or viable images aren't detected the server defaults to the Inventree logo/favicon location

**Reply by matmair** [reply]
We are happy to review PRs.
